src/docs/Doxyfile.9

```diff
@@ -81,17 +81,6 @@ OUTPUT_DIRECTORY       = ../../docs
 
 CREATE_SUBDIRS         = NO
 
-# Controls the number of sub-directories that will be created when
-# CREATE_SUBDIRS tag is set to YES. Level 0 represents 16 directories, and every
-# level increment doubles the number of directories, resulting in 4096
-# directories at level 8 which is the default and also the maximum value. The
-# sub-directories are organized in 2 levels, the first level always has a fixed
-# numer of 16 directories.
-# Minimum value: 0, maximum value: 8, default value: 8.
-# This tag requires that the tag CREATE_SUBDIRS is set to YES.
-
-CREATE_SUBDIRS_LEVEL   = 8
-
```

Comment:
The code changes look OK, but what's the deal with removing the config? Did that get removed or something?

Reply:
These are config items that were added post 1.9.1 and so our 1.9.1 doxygen doesn't recognise them. For example here's the `s_docs` trace if we don't have these changes:
```
####################### MESSAGE ############################
s_all run of: "./s_docs" resulted in:
    warning: ignoring unsupported tag 'CREATE_SUBDIRS_LEVEL' at line 93, file -
    warning: ignoring unsupported tag 'SHOW_HEADERFILE' at line 669, file -
    warning: ignoring unsupported tag 'WARN_IF_INCOMPLETE_DOC' at line 887, file -
    warning: ignoring unsupported tag 'WARN_LINE_FORMAT' at line 927, file -
    warning: ignoring unsupported tag 'INPUT_FILE_ENCODING' at line 970, file -
    warning: ignoring unsupported tag 'FORTRAN_COMMENT_AFTER' at line 1171, file -
    warning: ignoring unsupported tag 'HTML_COLORSTYLE' at line 1382, file -
    warning: ignoring unsupported tag 'DOCSET_FEEDURL' at line 1483, file -
    warning: ignoring unsupported tag 'FULL_SIDEBAR' at line 1694, file -
    warning: ignoring unsupported tag 'OBFUSCATE_EMAILS' at line 1725, file -
    warning: ignoring unsupported tag 'MATHJAX_VERSION' at line 1773, file -
    warning: ignoring unsupported tag 'DOT_COMMON_ATTR' at line 2495, file -
    warning: ignoring unsupported tag 'DOT_EDGE_ATTR' at line 2504, file -
    warning: ignoring unsupported tag 'DOT_NODE_ATTR' at line 2512, file -
    warning: ignoring unsupported tag 'DIR_GRAPH_MAX_DEPTH' at line 2662, file -
#######################
```

The top of the doxyfile says `# Doxyfile 1.9.5` so I expect we ran `doxygen -u Doxyfile.9` with a 1.9.5 binary and they were auto -generated for completeness.

Reply:
That makes sense now - thanks for the explanation.